content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1564 @@
+                TimerMode::Once,
+            );
+            // Will transition over half a second to the attack animation
+            animation_player
+                .play_with_transition(animations.attack.clone(), Duration::from_secs_f32(0.5));
+        }
+        if timer.0.tick(time.delta()).just_finished() {
+            // Once the attack animation is finished, restart the run animation
+            animation_player
+                .play_with_transition(animations.run.clone(), Duration::from_secs_f32(0.5))
+                .repeat();
+        }
+    }
+}
+```
+
+[`AnimationPlayer`]: https://docs.rs/bevy/0.10.0/bevy/animation/struct.AnimationPlayer.html
+[`play_with_transition`]: https://docs.rs/bevy/0.10/bevy/animation/struct.AnimationPlayer.html#method.play_with_transition
+
+### AcessKit integration into `bevy_ui`
+
+<div class="release-feature-authors">authors: @ndarilek</div>
+
+Games are for everyone: and the way they're built should reflect that.
+Accessible games are rare, and proper support is often an afterthought, both at an engine and a game level.
+By building our UI solution with accessibility in mind, we hope to fix that.
+
+Bevy has [joined `egui`](https://github.com/emilk/egui/pull/2294) in making the first steps towards cross-platform accessibility-by-default, with the help of the outstanding [AccessKit](https://github.com/AccessKit/accesskit) crate.
+To our knowledge, this makes Bevy the first general purpose game engine with first-party accessibility support.
+
+We've exposed Bevy's UI hierarchy and text elements to screen readers and other assistive devices, managed by the new on-by-default `bevy_a11y` crate.
+This is ultimately powered by the new [`AccessibilityNode`](https://dev-docs.bevyengine.org/bevy/a11y/struct.AccessibilityNode.html) component, which combines with the existing hierarchy to expose this information directly to AccessKit and the [`Focus`](https://dev-docs.bevyengine.org/bevy/a11y/struct.Focus.html) resource, which stores the entity that has keyboard focus.
+
+There's still a lot more to be done here: integrating the focus system with a [gamepad-driven UI controls](https://github.com/bevyengine/rfcs/pull/41) solution, cleaning up the data model to [make sure "accessible by default" is a reality](https://github.com/bevyengine/bevy/issues/7862)), and adding support for remaining features in AccessKit.
+
+Special thanks to `@mcwcampbell`, the lead author of AccessKit, for reviewing our integration and working with us to reduce the number of dependencies upstream, [substantially improving both compile times and final executable size](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1440978453). This is [still a serious challenge on Linux](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1432144117), and so the `accesskit_unix` feature flag is [disabled by default for now](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1433896811).
+
+## What's Next?
```

Comment:
For upcoming changes, TAA https://github.com/bevyengine/bevy/pull/7291 and SSAO https://github.com/bevyengine/bevy/pull/7402 are two big ones.